cms/envs/aws.py

```diff
@@ -283,6 +283,17 @@
 DATABASES = AUTH_TOKENS['DATABASES']
 
 MODULESTORE = convert_module_store_setting_if_needed(AUTH_TOKENS.get('MODULESTORE', MODULESTORE))
+
+MODULESTORE_FIELD_OVERRIDE_PROVIDERS = ENV_TOKENS.get(
+    'MODULESTORE_FIELD_OVERRIDE_PROVIDERS',
+    MODULESTORE_FIELD_OVERRIDE_PROVIDERS
+)
+
+XBLOCK_FIELD_DATA_OVERRIDE_CLASSES = ENV_TOKENS.get(
+    'XBLOCK_FIELD_DATA_OVERRIDE_CLASSES',
+    XBLOCK_FIELD_DATA_OVERRIDE_CLASSES
```

Comment:
Sure, once I get imports using the `path.to.module:Class.method` syntax working. At the moment, these are paths to classes, but they'll become paths to wrapper methods.